webapp/store/views.py

```diff
@@ -231,6 +235,30 @@ def brand_search_snap():
             status_code,
         )
 
+    def _get_file(file):
+        try:
+            with open(
+                os.path.join(flask.current_app.root_path, file), "r"
+            ) as stream:
+                data = yaml.load(stream)
+        except Exception:
+            data = None
+
+        return data
+
+    @store.route("/publisher/<publisher>")
+    def publisher_details_jetbrains(publisher):
+        """
+        A view to display the publisher details page for specific publisher.
+        """
+
+        context = _get_file("store/content/publishers/" + publisher + ".yaml")
```

Comment:
Is it safe to pass `publisher` directly from URL to file path without any regexp in route or any other validation?

Reply:
this is an excellent question. Let me check.